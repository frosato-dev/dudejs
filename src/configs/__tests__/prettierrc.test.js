import prettierConfig from "../prettierrc";

it("returns expected prettier configuration", () => {
  expect(prettierConfig).toMatchSnapshot();
});
