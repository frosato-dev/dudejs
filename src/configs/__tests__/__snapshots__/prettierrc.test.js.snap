// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns expected prettier configuration 1`] = `
Object {
  "bracketSpacing": false,
  "jsxBracketSameLine": false,
  "printWidth": 80,
  "semi": false,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false,
}
`;
